source/common/protobuf/utility.cc

```diff
@@ -656,8 +657,11 @@ void MessageUtil::unpackTo(const ProtobufWkt::Any& any_message, Protobuf::Messag
                                          any_message_with_fixup.DebugString()));
       }
       Config::VersionConverter::annotateWithOriginalType(*earlier_version_desc, message);
-      MessageUtil::onVersionUpgradeDeprecation(any_full_name);
-      return;
+      auto itr = exclude_v2_protos.find(any_full_name);
```

Comment:
It would be cleaner to move this logic inside `Config::ApiTypeOracle::getEarlierVersionDescriptor`. That way, you can be sure that anyone trying to find an earlier version descriptor of those protos we've deny listed will not find an earlier version descriptor.

Reply:
sure. will do it